File: Programação Orientada a Objetos/Códigos/Códigos/POO 4/Pessoas/PessoaFisica.php
<?php
require_once 'Pessoa.php';
class PessoaFisica extends Pessoa {
	private $cpf;
	private $ano_nasc;

	public function getCPF(){
		return $this->cpf;
	}

	public function setCPF($cpf): void{
		$this->cpf = $cpf;
	}

	public function getAno(){
		return $this->ano_nasc;
	}

	public function setAno($ano_nasc): void{
		$this->ano_nasc = $ano_nasc;
	}

}
?>

File: Object-Oriented Programming/Códigos/Códigos/POO 2/Projeto Pessoas/PessoaFisica.java
package Proj_Pessoas;

public class PessoaFisica extends Pessoa{
	private String cpf;
	public int ano_nasc;
	
	public PessoaFisica() {
		
	}
	
	public String getCpf() {
		return cpf;
	}

	public void setCpf(String cpf) {
		this.cpf = cpf;
	}

	public double getAno() {
		return ano_nasc;
	}

	public void setAno(int ano_nasc) {
		this.ano_nasc = ano_nasc;
	}
	
}
